.github/workflows/opex_api_services_cms.yml

```diff
@@ -13,6 +13,10 @@ on:
 
 env:
   API_NAME: api_services_cms
+  ARM_CLIENT_ID: ${{ secrets.ARM_CLIENT_ID }}
```

Comment:
The `ARM_CLIENT_ID` is set at environment scope and not at the entire repository, so it should be moved to line 29 to make it works